features/step_definitions/admin_steps.rb

```diff
@@ -70,6 +70,20 @@ def update_settings(settings)
   Language.find_or_create_by_short_and_name("FR", "Francais")
 end
 
+Given /^caching is enabled$/ do
```

Comment:
I think these changes might have snuck in from somewhere else? It doesn't look like they're being used in the cukes.